DopplerCustomDomain/CustomDomainProvider/CustomDomainProviderService.cs

```diff
@@ -0,0 +1,35 @@
+using Flurl.Http;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Tavis.UriTemplates;
+
+
+namespace DopplerCustomDomain.CustomDomainProvider
+{
+    public class CustomDomainProviderService : ICustomDomainProviderService
+    {
+        private readonly ILogger<CustomDomainProviderService> _logger;
+
+        public CustomDomainProviderService(ILogger<CustomDomainProviderService> logger)
+        {
+            _logger = logger;
+        }
+
+        public Task<List<string>> CreateAllCustomDomains()
+        {
+            throw new System.NotImplementedException();
+        }
+
+        public Task CreateCustomDomains(string domain)
```

Comment:
If I am not wrong, we should indicate what service should deal with the created domain, right?

![image](https://user-images.githubusercontent.com/1157864/88708075-7c531b00-d0e9-11ea-90fa-a993cd650350.png)

So, if I am not wrong, we will need another parameter.